common/environment.go

```diff
@@ -115,6 +117,29 @@ func (d *AutoLoginType) Parse(s string) error {
 	return err
 }
 
+// MarshalJSON customizes the JSON encoding for AutoLoginType
+func (d AutoLoginType) MarshalJSON() ([]byte, error) {
+	return json.Marshal(d.String())
+}
+
+// UnmarshalJSON customizes the JSON decoding for AutoLoginType
+func (d *AutoLoginType) UnmarshalJSON(data []byte) error {
+	var v interface{}
+	if err := json.Unmarshal(data, &v); err != nil {
+		return err
+	}
+	if strValue, ok := v.(string); ok {
+		return d.Parse(strValue)
+	}
+	// Handle numeric values
+	if numValue, ok := v.(float64); ok {
```

Comment:
why parse the number as a float64 if only uint8s are allowed for AutoLoginType?

Reply:
In Go, JSON numbers are decoded into float64 by default. Therefore, during JSON unmarshalling, we cannot directly check for uint8 type. Instead, we need to handle float64 and then ensure that the value fits within the range of uint8.